src/components/ApplyForm/ApplyForm.jsx

```diff
@@ -1,8 +1,139 @@
 import Button from "../../components/Buttton/Button";
+import { useState } from 'react'
+import { redirect, useNavigate } from 'react-router-dom';
+import Dropdown from "../Dropdown/Dropdown";
+import postCreateAccount from "../../api/post_create_account";
+import postLogin from "../../api/post_login";
 import "./ApplyForm.css";
-import Dropdown from "../../components/Dropdown/Dropdown";
+
 
 const ApplyForm = () =>{
+    const navigate = useNavigate();
+
+
+    const [errorMessage, setErrorMessage] = useState("")
+    const [formInvalid, setFormInvalid] = useState("")
+    const [checkedState, setCheckedState] = useState([]);
+    const [mentored, setMentored] = useState(true);
+
+    // until we have a separate get request for skills list:
+    const skills = [ "Python", "Django", "DRF", "React", "Javascript", "Frontend",  "Backend", "HTML-CSS"]
+
+    const [signupdetails, setSignupDetails] = useState({
+        first_name: "",
+        last_name: "",
+        email: "",
+        mobile: "",
+        social_account: "",
+        linkedin_account: "", 
+        github_profile: "",
+        username: "",
+        password: "",
+        has_mentored:false,
+        location: "",
+        skills: [],
+    })
+
+    const handleChange = (event) => {
+
+            const {id, value} = event.target;
+            setSignupDetails((prevDetails) => ({
+                ...prevDetails,
+                [id]: value,
+            }))
+            
+
+    }
+
+    const handleCheckboxChange = (event) => {
```

Comment:
This doesn't handle a value being unticked - once it's been ticked the skill is added to the POST request data, even if you untick it

Reply:
Upon further review, it does remove a value, but not the skill that aligns with the text against the item being ticked/unticked